rootfs/scheduler/mock.py

```diff
@@ -57,6 +57,18 @@ def stop(self, name):
         job['state'] = JobState.down
 
     # Related to k8s services
+    def resolve_state(self, pod):
+        # See "Pod Phase" at http://kubernetes.io/v1.1/docs/user-guide/pod-states.html
+        states = {
+            "Pending": JobState.initialized,
```

Comment:
the pod phases come from here: https://github.com/kubernetes/kubernetes/blob/b7dc1175ba64ce36a52fa44910ea7dcb283ffed9/pkg/api/types.go#L901-L919

I believe these are distinctly different compared to namespace phases: https://github.com/kubernetes/kubernetes/blob/4ca66d2aefa20c27b670b2fa890052daadc05294/docs/design/namespaces.md#phases